src/rootfs/files/configs/usr/local/bin/swarm-init.sh

```
@@ -0,0 +1,227 @@
+            RELEASE_TAG="$HOST_AGENT_TAG"
+        elif [[ "$HOST_AGENT_TAG" == host-agent-* ]]; then
+            VERSION="${HOST_AGENT_TAG#host-agent-}"
+            RELEASE_TAG="release-$VERSION"
+        else
+            RELEASE_TAG="release-$HOST_AGENT_TAG"
+        fi
+        FILENAME="swarm-host-agent-${RELEASE_TAG}-linux-amd64.tar.gz"
+        TMP=$(mktemp -d)
+        download_github_asset "Super-Protocol" "swarm-cloud" "$RELEASE_TAG" "$FILENAME" "$TMP/host-agent.tar.gz"
+        tar xzf "$TMP/host-agent.tar.gz" -C "$TMP"
+        EXTRACT_DIR=$(ls -1 "$TMP" | grep -v 'host-agent\.tar\.gz' | head -1)
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)